lib/active_admin/view_helpers/breadcrumb_helper.rb

```diff
@@ -19,7 +20,7 @@ def breadcrumb_links(path = request.path)
             name = display_name config.find_resource part
           end
           name ||= I18n.t "activerecord.models.#{part.singularize}", count: ::ActiveAdmin::Helpers::I18n::PLURAL_MANY_COUNT, default: part.titlecase
-
+          name = CGI.unescape name
```

Comment:
What's the reasoning for using `CGI.unescape` on the `name` as well and not just the `part` value?